### DISCUSSION TITLE
Anh em dùng của bác nhiều lắm cơ mà ít sao nhở

### DISCUSSION THREAD

**Discussion by nonqxt86** [abpvn/abpvn#526] [General]
Em vừa cho bác một sao. Chúc bác luôn mạnh khỏe và vui vẻ

**Comment by hoang-rio** [comment]
Vì ABPVN tích hợp sẵn ở các tiện ích chặn quảng cáo rồi nên mọi người cũng k biết là có repo này luôn ấy bạn
